server/plugin.go

```diff
@@ -314,6 +316,52 @@ func (p *Plugin) getZoomUserWithToken(token *oauth2.Token) (*zoom.User, error) {
 	return &zoomUser, nil
 }
 
+func (p *Plugin) GetMeetingOAuth(meetingID int, userID string) (*zoom.Meeting, error) {
+	config := p.getConfiguration()
+	ctx := context.Background()
+
+	conf, err := p.getOAuthConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	zoomUserInfo, apiErr := p.getZoomUserInfo(userID)
+	if apiErr != nil {
+		return nil, err
+	}
+
+	client := conf.Client(ctx, zoomUserInfo.OAuthToken)
+	apiURL := config.ZoomAPIURL
+	if apiURL == "" {
+		apiURL = zoomDefaultAPIURL
+	}
+
+	url := fmt.Sprintf("%v/meetings/%v", apiURL, meetingID)
+	res, err := client.Get(url)
+	if err != nil || res == nil {
+		return nil, errors.New("error fetching zoom user, err=" + err.Error())
```

Comment:
if it's the case that `err == nil && res == nil`, this line will panic. I'm not sure if that's possible, but best to avoid panics when possible.